components/html-block/test/html-block.vdiff.js

```diff
@@ -0,0 +1,308 @@
+import '../html-block.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+
+const content = `
+	Just a text node...
+	<h1>heading 1</h1>
+	<h2>heading 2</h2>
+	<h3>heading 3</h3>
+	<h4>heading 4</h4>
+	<h5>heading 5</h5>
+	<h6>heading 6</h6>
+	<div><strong>strong</strong></div>
+	<div><b>bold</b></div>
+	<div>text</div>
+	<pre>preformatted</pre>
+	<p>paragraph</p>
+	<ul>
+		<li>unordered item 1</li>
+		<li>unordered item 2</li>
+	</ul>
+	<ol>
+		<li>ordered item 1</li>
+		<li>ordered item 2</li>
+	</ol>
+	<div><a href="https://d2l.com">anchor</a></div>
+	<p dir="rtl">rtl paragraph</p>
+`;
+
+const mathBlock = `
+	<math xmlns="http://www.w3.org/1998/Math/MathML" display="block">
+		<msup>
+			<mrow>
+				<mo>(</mo>
+				<mrow>
+					<munderover>
+						<mo>∑<!-- ∑ --></mo>
+						<mrow class="MJX-TeXAtom-ORD">
+							<mi>k</mi>
+							<mo>=</mo>
+							<mn>1</mn>
+						</mrow>
+						<mi>n</mi>
+					</munderover>
+					<msub>
+						<mi>a</mi>
+						<mi>k</mi>
+					</msub>
+					<msub>
+						<mi>b</mi>
+						<mi>k</mi>
+					</msub>
+				</mrow>
+				<mo>)</mo>
+			</mrow>
+			<mrow class="MJX-TeXAtom-ORD">
+				<mspace width="negativethinmathspace"></mspace>
+				<mspace width="negativethinmathspace"></mspace>
+				<mn>2</mn>
+			</mrow>
+		</msup>
+		<mo>≤<!-- ≤ --></mo>
+		<mrow>
+			<mo>(</mo>
+			<mrow>
+				<munderover>
+					<mo>∑<!-- ∑ --></mo>
+					<mrow class="MJX-TeXAtom-ORD">
+						<mi>k</mi>
+						<mo>=</mo>
+						<mn>1</mn>
+					</mrow>
+					<mi>n</mi>
+				</munderover>
+				<msubsup>
+					<mi>a</mi>
+					<mi>k</mi>
+					<mn>2</mn>
+				</msubsup>
+			</mrow>
+			<mo>)</mo>
+		</mrow>
+		<mrow>
+			<mo>(</mo>
+			<mrow>
+				<munderover>
+					<mo>∑<!-- ∑ --></mo>
+					<mrow class="MJX-TeXAtom-ORD">
+						<mi>k</mi>
+						<mo>=</mo>
+						<mn>1</mn>
+					</mrow>
+					<mi>n</mi>
+				</munderover>
+				<msubsup>
+					<mi>b</mi>
+					<mi>k</mi>
+					<mn>2</mn>
+				</msubsup>
+			</mrow>
+			<mo>)</mo>
+		</mrow>
+		<mspace linebreak="newline"></mspace>
+		<msup>
+			<mi>e</mi>
+			<mrow>
+				<mi>i</mi>
+				<mi>π<!-- π --></mi>
+			</mrow>
+		</msup>
+		<mo>+</mo>
+		<mn>1</mn>
+		<mo>=</mo>
+		<mn>0</mn>
+	</math>
+`;
+
+const code = `<pre class="d2l-code d2l-code-dark"><code class="language-javascript">function helloGrumpy(name) {
+	console.log(\`Hi there \${name}.\`);
+}
+helloGrumpy('Wizard');</code></pre>`;
+
+describe('d2l-html-block', () => {
+
+	const viewport = { width: 376 };
```

Comment:
This weird value is to account for the different page padding/margins in the new framework.